packages/client/src/components/form/DocumentUploadField/DocumentUploaderWithOption.tsx

```diff
@@ -153,14 +153,17 @@ export const DocumentUploaderWithOption = (props: IFullProps) => {
       options.maxSizeMB = props.compressImagesToSizeMB
     }
     // disable compression with a falsy value
-    const resized =
+    const resized: Blob =
       Boolean(options.maxSizeMB) &&
       bytesToMB(uploadedImage.size) > options.maxSizeMB &&
       (await imageCompression(uploadedImage, options))
 
     const fileAsBase64 = await getBase64String(resized || uploadedImage)
 
-    return fileAsBase64.toString()
+    return {
+      fileAsBase64: fileAsBase64.toString(),
+      resized
+    }
```

Comment:
@rikukissa ,
Since the function is only used in one place for now and **both formats are needed** here: 

```
 const newDocument: IFileValue = {
      optionValues,
      type: resizedImage.type || uploadedImage.type,
      data: fileAsBase64.toString(),
      fileSize: resizedImage.size || uploadedImage.size
    }
```

I think we can keep both **OR** I suggest adding an optional `withCompressed` parameter to the `processImage` function. 
This way, we can return:
`return withCompressed ? { fileAsBase64: fileAsBase64.toString(), resized } : fileAsBase64.toString();
`
What do you think?